static/styles/portico/portico_signin.css

```diff
@@ -232,7 +232,6 @@ html {
 .header {
     color: hsl(0, 0%, 27%);
     background-color: hsl(0, 0%, 100%);
-    position: fixed;
```

Comment:
This is a far-reaching CSS change; what's your thinking on why this is correct and won't create problems with other pages using this class?

(If this is just deleting a duplicate parameter, or something, please do that in a separate commit with a comimt message explaining the reasoning)